contrib/examples/walkthrough/ups-instance.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: test-ns
 spec:
   serviceClassName: user-provided-service
-  planName: default
+  planName: default86064792-7ea2-467b-af93-ac9694d96d52
```

Comment:
really really really want a dash between the service name and plan id - there's ugly and there's annoying - we voted for ugly  :-)

Reply:
actually, is this really right even w/o the dash?  Shouldn't it be "user-provided-service-default...." ?

Reply:
too bad

Reply:
Is this a blocker for you? I think it's a waste of anybody's time to go through and make this change.

Reply:
I'd like the PR to match what we agreed to on the last call

Reply:
I don't see that particular detail in the consensus agreement. I'll go do this. It is a waste of a couple of hours for something that's changing anyway.